Makefile

```diff
@@ -12,6 +12,8 @@ PROJECT_ROOT=github.com/superfly/wormhole
 # Race detector is only supported on amd64.
 RACE := $(shell test $$(go env GOARCH) != "amd64" || (echo "-race"))
 
+MUTEX := $(shell test $$(go env GOARCH) != "amd64" || (echo "-mutexprofile=mutex.out"))
```

Comment:
is this being used in any makefile target?

Reply:
Ah no. Was going to have it be part of an extended test target for debugging. But was going to be for a future PR if I ever merged - must have slipped through. I'll trash this

Reply:
👍

Reply:
let's remove this before merging